src/server/routes/entity/edition.ts

```diff
@@ -275,9 +271,14 @@ router.get(
 			}));
 		}
 
-		makePromiseFromObject(propsPromise)
-			.then(render)
-			.catch(next);
+		try {
+			const props = await makePromiseFromObject(propsObject);
```

Comment:
I think `makePromiseFromObject` is not needed anymore here, as it was used to await all the promises added to propsPromise.
Since we are now directly awaiting each value I think the utility doesn't do anything.